guide/english/3d/blender/index.md

```diff
@@ -6,11 +6,11 @@ title: Blender
 
 ## Blender
 
-Blender is a free and open source 3D creation suite. It's used for modeling, creating photorealistic renders, simulations, model sculpting, architecture models, compositing, character design, animation, rigging, level design, 2D rendered scenes, motion graphics, game development, VFX, video editing, and even much more!
+Blender is a free and open source 3D creation suite. It's used for modelling, creating photorealistic renders, simulations, model sculpting, architecture models, compositing, character design, animation, rigging, level design, 2D rendered scenes, motion graphics, game development, VFX, video editing, and even much more!
 
 The Blender software is cross-platform that runs on Windows, Linux, and macOS. Blender is supported by The Blender Foundation, a public-benefit organization from the Netherlands. Developers, animators, scientists, experts and even hobbyists empower the software's development by constantly maintaining, fixing and making new features for a suitable and more responsive user experience.
 
-Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as an .obj that slicers can convert to .stl files or it can export in .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to an .stl - a very popular filetype.
+Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files or it can export in a .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.
```

Comment:
```suggestion
Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files ,it can export in the .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.
```

Reply:
Please make this change too